build/common.mk

```diff
@@ -109,7 +109,7 @@ GO_TEST_PKG_PARALLELISM ?= 2    # -p flag, number of parallel packages to test
 GO_TEST_SHUFFLE         ?= off  # -shuffle flag, randomizes order of tests within a package
 GO_TEST_TAGS            ?= all
 
-GO_TEST_FLAGS = -count=1 -cover -tags="${GO_TEST_TAGS}" -timeout 1h \
+GO_TEST_FLAGS = -count=1 -cover -tags="${GO_TEST_TAGS}" -timeout 2h \
```

Comment:
Bit concerning, some perf issue with 1.50?

Reply:
Not exactly with 1.50, but with how we handle dependencies / venvs in integration tests.  There's a slack thread I'll ping you in, but the tl;dr is:
- every single test creates its own venv
- every venv installs both the released version of pulumi and the local version
- grpcio does not ship wheels for M1 macs, so it's built from source twice for every single test
#11485 was opened as an actual solution to this, so I'll be reverting this change here.

Reply:
Understood